geode-core/src/main/java/org/apache/geode/internal/cache/BucketRegion.java

```diff
@@ -235,6 +237,12 @@ AtomicLong5 getEventSeqNum() {
     return eventSeqNum;
   }
 
+  boolean notPrimary = true;
+  volatile boolean allChildBucketsBecomePrimary = false;
```

Comment:
All access to `allChildBucketsBecomePrimary` is done under synchronization so `volatile` is redundant.